pymc_experimental/tests/test_distributions.py

```diff
@@ -0,0 +1,61 @@
+#   Copyright 2020 The PyMC Developers
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+import functools
+import itertools
+import sys
+
+import aesara
+import aesara.tensor as at
+import numpy as np
+import numpy.random as nr
+
+import pytest
+import scipy.stats
+import scipy.stats.distributions as sp
```

Comment:
`sp` is an odd alias (if you even need it). Usually used for `scipy.special`

Reply:
This is in the main `pymc` test code (https://github.com/pymc-devs/pymc/blob/1005d20b3c12d9b9a424c069f6a0f9962d73c41d/pymc/tests/test_distributions.py#L47) as such and used by AR1: https://github.com/pymc-devs/pymc/blob/main/pymc/tests/test_distributions.py#L502

Reply:
Indeed... seems like we should change it over there as well. Feel free to ignore my comment in the meantime :D

Reply:
Changed to ssd!